fetch.bs

```diff
@@ -1296,6 +1296,8 @@ downloads. This format of range header value can be set using <a>add a range hea
 <a>implementation-defined</a> <a for=/>header value</a> for the `<code>User-Agent</code>`
 <a for=/>header</a>.
 
+<p>The <dfn>Document `Accept` header value</dfn> is
```

Comment:
```suggestion
<p>The <dfn>document `<code>Accept</code>` header value</dfn> is
```
(I guess we should rename the other one to use "header value" as well in a separate PR.)

Reply:
I'm not sure how this was resolved?

Reply:
Apologies, I think I might have overwritten some previous commit. Will fix